code/modules/paperwork/pen.dm

```diff
@@ -137,17 +137,24 @@
  */
 /obj/item/pen/edagger
 	origin_tech = "combat=3;syndicate=1"
-	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut") //these wont show up if the pen is off
 	var/on = 0
 	var/brightness_on = 2
 	light_color = LIGHT_COLOR_RED
 
+/obj/item/pen/edagger/attack(mob/living/M, mob/living/user, def_zone)
+	if(on == 1 && user.dir == M.dir && !M.weakened && user != M)
```

Comment:
```suggestion
	if(on && user.dir == M.dir && !M.incapacitated(TRUE) && user != M)
```
using `incapacitated` will mean that you also don't backstab when they are resting or sleeping etc. the `TRUE` is there to avoid restraints from blocking it.
`on` is also a boolean value